### DISCUSSION TITLE
Could not initialize OpenSSL library

### DISCUSSION THREAD

**Discussion by BorbaClaudio** [viniciussanchez/RESTRequest4Delphi#206] [General]
Olá, estou usando o Lazarus 3.0 e Linux Mint 21.3, usando o exemplo RESTRequest4Delphi/samples/client/lazarus esta dando a mensagem de erro: Could not initialize OpenSSL library.

**Comment by BorbaClaudio** [comment]
Consegui abrir a bliblioteca OpenSSL, agora outro erro:
Unexpected response status code: 404

**Reply by viniciussanchez** [reply]
Seria interessante você colocar também como resolveu o problema anterior, para caso alguém tenha a mesma dificuldade, ter onde pesquisar...
